antgroup/deploy/ray-operator/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.13.8 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN GO111MODULE=on GOPROXY=https://goproxy.io go mod download
```

Comment:
Maybe don't need GOPROXY in open source

Reply:
I consider to add it not only for oversea users also for domestic users.